azext_edge/edge/_help.py

```diff
@@ -1086,3 +1086,60 @@ def load_iotops_help():
             az iot ops asset endpoint certificate remove --endpoint {asset_endpoint} -g {resource_group}
             --thumbprint {thumbprint}
     """
+
+    helps[
+        "iot ops schema"
+    ] = """
+        type: group
+        short-summary: Schema management.
+    """
+
+    helps[
+        "iot ops schema registry"
+    ] = """
+        type: group
+        short-summary: Schema registry management.
+    """
+
+    helps[
+        "iot ops schema registry show"
+    ] = """
+        type: command
+        short-summary: Show details of a schema registry.
+        examples:
+        - name: Show details of target schema registry 'myregistry'.
+          text: >
+            az iot ops schema registry show --name myregistry -g myresourcegroup
+    """
+
+    helps[
+        "iot ops schema registry list"
+    ] = """
+        type: command
+        short-summary: List schema registries in a resource group or subscription.
+        examples:
+        - name: List schema registeries in the resource group 'myresourcegroup'.
+          text: >
+            az iot ops schema registry list -g myresourcegroup
+        - name: List schema registeries in the default subscription filtering on a particular tag.
+          text: >
+            az iot ops schema registry list --query "[?tags.env == 'prod']"
+    """
+
+    helps[
+        "iot ops schema registry delete"
+    ] = """
+        type: command
+        short-summary: Delete a target schema registry.
+        examples:
+        - name: Delete schema registry 'myregistry'.
+          text: >
+            az iot ops schema registry delete -n myregistry -g myresourcegroup
+    """
+
+    helps[
+        "iot ops schema registry create"
+    ] = """
+        type: command
+        short-summary: Create a schema registry.
```

Comment:
example plz with the storage uri (you can prob make up the storage account + container names)

Reply:
Please read the PR description for more info on this.

Reply:
ok... but put it into your todo wherever you keep them